ui/src/shared/components/cloud/CloudExclude.tsx

```diff
@@ -0,0 +1,13 @@
+import {PureComponent} from 'react'
+
+export default class extends PureComponent {
```

Comment:
Theres already a component called cloud feature flag that does basically this. should that component be used rather than creating anew one?